util/command_manager.js

```diff
@@ -1,34 +1,46 @@
 const CommandList = require("./CommandList");
+function resolveAll(arr){
+    return Promise.all(arr.map(x=>new Promise(r=>x.catch(_=>console.log("Failed doing Something.")).then(r))))
```

Comment:
Shouldn't you have access to the error somewhere in here? Pretty sure there's something better than "Failed doing Something." we can return.

Reply:
Yes, but it is most probably a fetch error which is multiple lines. Ideally it'd say "failed pushing/deleting (command)" but that needs some refactoring, so I put that as a placeholder

Reply:
> Shouldn't you have access to the error somewhere in here? Pretty sure there's something better than "Failed doing Something." we can return.



> Merging for now, we should probably separate command syncing utils from bot utils at some point

Yeah